src/Persistence_matrix/include/gudhi/persistence_interval.h

```diff
@@ -0,0 +1,129 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2024 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+/**
+ * @file persistence_interval.h
+ * @author Hannah Schreiber
+ * @brief Contains @ref Gudhi::persistence_matrix::Persistence_interval class.
+ */
+
+#ifndef PM_INTERVAL_INCLUDED
+#define PM_INTERVAL_INCLUDED
+
+#include <iostream>
```

Comment:
```suggestion
#include <ostream>
```
would be sufficient?